### DISCUSSION TITLE
How to use configuration files

### DISCUSSION THREAD

**Discussion by kN6jq** [projectdiscovery/uncover#3] [Q&A] [answered]
How to use configuration files？

![image](https://user-images.githubusercontent.com/27048404/156486148-22dd1299-c6a5-471b-8aea-0bd6b8ae0584.png)

documentation means
shodan:
  - SHODAN_API_KEY1
  - SHODAN_API_KEY2
censys:
  - CENSYS_API_ID:CENSYS_API_SECRET
fofa:
  - FOFA_EMAIL:FOFA_KEY

but my yaml file is 
shodan: []
censys: []
fofa: []

Can you give a test example？thiks

**Comment by ehsandeep** [comment, marked as answer]
@kN6jq By default, a blank provider configuration file is created, you can update it using the format described in the readme, i.e

```
shodan:
  - XXXXXX
```
